UI/api-interface.cpp

```diff
@@ -620,6 +620,11 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return bstrdup(recordOutputPath);
 	}
 
+	char *obs_frontend_get_locale_string(const char *string) override
+	{
+		return bstrdup(Str(string));
```

Comment:
There's probably no need to duplicate the string. Pretty sure the original pointer is safe in terms of lifetime. This whole thing should probably not duplicate, and the return should probably be const.